libs/block-template-react/src/base.module.scss

```
@@ -1,251 +1,45 @@
-  --tertiary-background-base: var(--neutral-100);
-  --tertiary-background-hover: var(--neutral-200);
-
-  /* brand colours */
-  --brand-primary-base: var(--primary-600);
-  --brand-primary-hover: var(--primary-700);
-  --brand-secondary-base: var(--secondary-600);
-  --brand-secondary-hover: var(--secondary-700);
-  --brand-tertiary-base: var(--tertiary-600);
-  --brand-tertiary-hover: var(--tertiary-700);
-
-  /* text colours */
-  --primary-text-base: var(--neutral-800);
-  --primary-text-hover: var(--neutral-900);
-  --secondary-text-base: var(--neutral-600);
-  --secondary-text-hover: var(--neutral-700);
-
-  /* border colours */
-  --border-color-light: var(--neutral-200);
-  --border-color-dark: var(--neutral-400);
-
-  /* spacing */
-  --spacing-base: 1rem;
-  --space-1: calc(var(--spacing-base) * 0.25);
-  --space-2: calc(var(--spacing-base) * 0.5);
-  --space-3: calc(var(--spacing-base) * 0.75);
-  --space-4: var(--spacing-base);
-  --space-5: calc(var(--spacing-base) * 1.5);
-  --space-6: calc(var(--spacing-base) * 2);
-  --space-7: calc(var(--spacing-base) * 3);
-  --space-8: calc(var(--spacing-base) * 4.5);
-
-  /* font sizes */
-  --font-size-xs: clamp(0.8rem, 0.17vw + 0.76rem, 0.89rem);
-  --font-size-sm: clamp(0.85rem, 0.22vw + 0.83rem, 0.95rem);
-  --font-size-base: clamp(1rem, 0.34vw + 0.91rem, 1.12rem);
-  --font-size-md: clamp(1.25rem, 0.61vw + 1.1rem, 1.58rem);
-  --font-size-lg: clamp(1.56rem, 1vw + 1.31rem, 2.11rem);
-  --font-size-xl: clamp(1.95rem, 1.56vw + 1.56rem, 2.81rem);
-  --font-size-2xl: clamp(2.44rem, 2.38vw + 1.85rem, 3.75rem);
-  --font-size-3xl: clamp(3.05rem, 3.54vw + 2.17rem, 5rem);
-
-  /* line heights */
-  --line-height-tight: 1.25;
-  --line-height-base: 1.5;
-  --line-height-loose: 1.75;
-
-  /* font family */
-  --font-family: system-ui, "Inter", "Open Sans", "Helvetica Neue", "Arial",
-    sans-serif;
-
-  /* border radius */
-  --border-radius-1: 0.25rem;
-  --border-radius-2: 0.5rem;
-  --border-radius-3: 1rem;
-
-  /* box shadow */
-  --shadow-1: 0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);
-  --shadow-2: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
-  --shadow-3: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
-  --shadow-4: 0 25px 50px -12px rgb(0 0 0 / 0.25);
-
-  /* border weight */
-  --border-weight-thin: 1px;
-  --border-weight-light: 2px;
-  --border-weight-base: 4px;
-
-  /* icon size */
-  --icon-size-base: 1rem;
-  --icon-size-large: 1.5rem;
+/*  ----------------------------------------------------------------------
+Styles used for the Hello World example included with block-react-template 
+----------------------------------------------------------------------  */
 
+.block {
   /* –––––––––– Helper styles –––––––––– */
 
-  font-family: var(--font-family);
-  color: var(--primary-text-base);
-
-  /* Typography */
+  font-family: system-ui, "Inter", "Open Sans", "Helvetica Neue", "Arial",
+    sans-serif;
```

We don't load these fonts so I'd just get rid of them
```suggestion
  font-family: system-ui, "Helvetica Neue", "Arial", sans-serif;
```
```

---

Done in 37b0a92d5d1fe8df92ccd12b4509ec0d28d6a8b3

---

See suggestion above

---

I misunderstood, thank you. 7b2f14a38eb4968cb6b76d2560a15656eea3a637